namd/tests/library/018_pathCV/AutoDiff/test.pmf

```diff
@@ -2,1220 +2,1220 @@
 # -5.25000000000000e-01  5.00000000000000e-02         41  0
 
- -2.00000000000000e-01 -5.00000000000000e-01   5.69586149951569e-02
- -2.00000000000000e-01 -4.50000000000000e-01   5.69382966644937e-02
- -2.00000000000000e-01 -4.00000000000000e-01   5.68769461750447e-02
- -2.00000000000000e-01 -3.50000000000000e-01   5.67733665267178e-02
- -2.00000000000000e-01 -3.00000000000000e-01   5.66255457146456e-02
- -2.00000000000000e-01 -2.50000000000000e-01   5.64306427548948e-02
- -2.00000000000000e-01 -2.00000000000000e-01   5.61849600753329e-02
- -2.00000000000000e-01 -1.50000000000000e-01   5.58840288715106e-02
- -2.00000000000000e-01 -1.00000000000000e-01   5.55227848234053e-02
- -2.00000000000000e-01 -5.00000000000000e-02   5.50959454137594e-02
- -2.00000000000000e-01  0.00000000000000e+00   5.45988989681669e-02
- -2.00000000000000e-01  5.00000000000000e-02   5.40294992011255e-02
- -2.00000000000000e-01  1.00000000000000e-01   5.33916589593868e-02
- -2.00000000000000e-01  1.50000000000000e-01   5.27022394910228e-02
- -2.00000000000000e-01  2.00000000000000e-01   5.20040670648128e-02
- -2.00000000000000e-01  2.50000000000000e-01   5.13897033729455e-02
- -2.00000000000000e-01  3.00000000000000e-01   5.10423626115960e-02
- -2.00000000000000e-01  3.50000000000000e-01   5.12985377781979e-02
- -2.00000000000000e-01  4.00000000000000e-01   5.27181985397260e-02
- -2.00000000000000e-01  4.50000000000000e-01   5.60782581523479e-02
- -2.00000000000000e-01  5.00000000000000e-01   6.20237943536309e-02
- -2.00000000000000e-01  5.50000000000000e-01   6.98823887905351e-02
- -2.00000000000000e-01  6.00000000000000e-01   7.60044966151296e-02
- -2.00000000000000e-01  6.50000000000000e-01   7.96239617900707e-02
- -2.00000000000000e-01  7.00000000000000e-01   8.13077153155626e-02
- -2.00000000000000e-01  7.50000000000000e-01   8.17953161839064e-02
- -2.00000000000000e-01  8.00000000000000e-01   8.16372048306354e-02
- -2.00000000000000e-01  8.50000000000000e-01   8.11731427050515e-02
- -2.00000000000000e-01  9.00000000000000e-01   8.05935862919719e-02
- -2.00000000000000e-01  9.50000000000000e-01   7.99984398126379e-02
- -2.00000000000000e-01  1.00000000000000e+00   7.94367189537887e-02
- -2.00000000000000e-01  1.05000000000000e+00   7.89301053676405e-02
- -2.00000000000000e-01  1.10000000000000e+00   7.84861755647861e-02
- -2.00000000000000e-01  1.15000000000000e+00   7.81055663777811e-02
- -2.00000000000000e-01  1.20000000000000e+00   7.77858043866154e-02
- -2.00000000000000e-01  1.25000000000000e+00   7.75232734973943e-02
- -2.00000000000000e-01  1.30000000000000e+00   7.73142286913480e-02
- -2.00000000000000e-01  1.35000000000000e+00   7.71553039111600e-02
- -2.00000000000000e-01  1.40000000000000e+00   7.70437678236428e-02
- -2.00000000000000e-01  1.45000000000000e+00   7.69776442679124e-02
- -2.00000000000000e-01  1.50000000000000e+00   7.69557395004421e-02
+ -2.00000000000000e-01 -5.00000000000000e-01   5.66747557569237e-02
```

Comment:
Why are the test data changed here? I know this PR fixes the boundary condition in the integrator but `018_pathCV` does not use any periodic CVs.

Reply:
The fix applies to Neumann (non-periodic) boundary conditions.